app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryEdit2Fragment.kt

```diff
@@ -316,6 +289,24 @@ class ContentEntryEdit2Fragment(private val registry: ActivityResultRegistry? =
             }
             loading = false
         }
+
+        navController.currentBackStackEntry?.savedStateHandle?.observeResult(this,
```

Comment:
This is to be moved to the core presenter...

Also: add a key - normally that would be set according to the result type. Because this is just a string, we should give it a keyname so it is clear.